app_dart/lib/src/request_handlers/append_log.dart

```diff
@@ -51,6 +66,13 @@ class AppendLog extends ApiRequestHandler<Body> {
       await transaction.commit();
     });
 
+    await writeToStackdriver('${encodedOwnerKey}_${task.attempts}');
```

Comment:
It would be more readable / convenient to have an example logName in comment.

Reply:
Added!